CredentialProvider/Cells/NoSearchResultCell.swift

```diff
@@ -7,14 +7,48 @@ import UIKit
 
 class NoSearchResultCell: UITableViewCell {
     
-    @IBOutlet weak var titleLabel: UILabel! {
-        didSet {
-            titleLabel.text = .LoginsListNoMatchingResultTitle
+    static let identifier = "noSearchResultCell"
+    
+    lazy private var titleLabel: UILabel = {
+        let label = UILabel()
+        label.textColor = UIColor(named: "labelColor")
```

Comment:
Can we extract colors like this into constants? Do we have them already?

Reply:
Ok, I will check and if we do not have them, I will extract them into constants, as you suggested. Thank you